### ISSUE TITLE
bB Movie Regex not detecting quality etc.

### ISSUE THREAD

**Issue by jseycom** [Jackett/Jackett#833]
HTML:

```html
<tr class="torrent  ">
		<td class="center"><a href="torrents.php?filter_cat[9]=1"><img src="static/common/caticons/movies.png" alt="Movies" title="Movies" width="24" height="24"></a></td>
		<td>
			<span>[<a href="torrents.php?action=download......" title="Download">DL</a> | <a href="reports......" title="Report">RP</a> ] </span>
			<a href="torrents.php......" title="View Torrent">Name</a>			 (2016) [BluRay / x264 / DTS / MKV / 1080p / Scene / w. Subtitles - <strong>Freeleech!</strong>]			<br><div class="tags"></div>		</td>
		<td>3</td>
		<td class="nobr">1 day, 7 hours</td>
		<td class="nobr">9.84 GB</td>
		<td>9</td>
		<td>44</td>
		<td>33</td>
		<td>3</td>
	</tr>
```

The problem is that only the text of the <a> is searched, but the quality information etc. is actually outside. This causes incorrect releases to be downloaded.

**Comment by kaso17** [comment]
Fixed with d259f3df0d6842540f8019824d14b7422fd17c0a

**Comment by jseycom** [comment]
Thanks @kaso17 

This seems to have fixed the searching, but now downloading gives a 404.

The logs show that the file is actually downloading from bB, but failing within Jackett. 

The URL Jackett gives for the download ends with `&file=%0A%09%09%09%0A%09%09%09Name+here%09%09%09+(1972)+%5BDVD5+%2F+x264+%2F+AC-3+%2F+MKV+%2F+SD+%2F+Scene+%2F+w+Subtitles%5D%09%09%09%09%09.torrent`

If I remove all that extra spacing around the name, it works fine.

**Comment by kaso17** [comment]
fixed with b4fafe2845dcbfe0069333e2f761b9f17c1048f4
